.github/workflows/pull-request.yml

```
@@ -113,26 +127,41 @@
-            INSTALLATION.md
-            ISSUE_GUIDELINES.md
-            PR_GUIDELINES.md
-            README.md
-            CODEOWNERS
-            LICENSE  
-            .github/**
-            analysis_options.yaml
-            pubspec.yaml
-            pubspec.lock
-            lib/main.dart
+        run : | 
```

YAMLlint (1.37.1)

[error] too many spaces before colon

(colons)